echo.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+  "net/http"
+  "github.com/labstack/echo"
+)
+
+func Hello(c echo.Context) error {
+  return c.String(http.StatusOK, "Hello ALTA Store Customer")
+}
+
+func main() {
+  e := echo.New()
+  e.Get("/hello", Hello)
```

Comment:
ini ada typo di saya mas Edo, waktu dijalanin jadi error karena function Get() tidak dikenali, seperti gambar di bawah

![Error method Get tidak dikenali](https://cdn.discordapp.com/attachments/750304684836651039/813252007225196574/unknown.png)

bisa minta tolong diganti jadi GET kapital semua mas? kalau GET nya kapital waktu saya tes di browser sudah bisa jalan dengan sesuai

![berjalan dengan baik](https://cdn.discordapp.com/attachments/750304684836651039/813251490650914876/unknown.png)

```suggestion
  e.GET("/hello", Hello)
```